content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,746 @@
+time off types, leave this field blank.
+
+Next, select how the :guilabel:`Level Transition` occurs, either :guilabel:`Immediately` or
+:guilabel:`After this accrual's period`. By default, the first level begins once the time off is
+approved if the time off is based on an accrual plan. If :guilabel:`Immediately` is selected, then
+the next level begins according to the time frame set on the level. If :guilabel:`After this
+accrual's period` is selected, the next level does not begin until the first level is completed
+according to the rules set on it.
+
+Rules
+*****
+
+Rules must be created in order for the accrual plan to accrue time off.
+
+To create a new rule, click the :guilabel:`Add A New Level` button right beneath the word `Rules`,
+and a :guilabel:`Create Level` pop-up form appears.
+
+Fill out the following fields on the form:
+
+- :guilabel:`Start after (#) (time period) after allocation date`: enter the number and value of the
+  time period that must pass before the employee starts to accumulate time off. The first value is
+  numerical; enter a number in the first field.
+
+  Then, select the type of time period using the drop-down menu in the second field. The options
+  are: :guilabel:`day(s)`, :guilabel:`month(s)`, or :guilabel:`year(s)`.
+- :guilabel:`Based on worked time`: tick this box if the accrual of time off is based on the
+  time the employee has worked. If an employee takes time off that is *not* considered a worked day,
+  Odoo will not count that day towards their accrual plan.
+- :guilabel:`Rate (#) (time)`: enter the rate of time off that is accumulated. The first value is
+  numerical; enter a number in the first field. Whole numbers are not necessary, any decimal value
+  may be entered.
+
+  Next, in the second field, select the type of time accrued using the drop-down menu. The options
+  are either :guilabel:`Days` or :guilabel:`Hours`.
+- :guilabel:`Frequency (X)`: select how often the employee accrues the time off for this rule using
+  the drop-down menu. The options are :guilabel:`Daily`, :guilabel:`Weekly`, :guilabel:`Twice a
+  month`, :guilabel:`Monthly`, :guilabel:`Twice a year`, or :guilabel:`Yearly`.
+
+  Depending on the selection, more fields appear to specify exactly when the accrual renews.
+
+.. example::
+   If the employee should accrue one vacation day for every week worked, the :guilabel:`Rate` is set
+   to `1`, and the :guilabel:`Frequency` entry is set to `Frequency (Weekly) on (Friday)`. Only the
+   :guilabel:`Frequency` and :guilabel:`Weekday` fields appear.
+
+   If the employee should accrue ten (10)vacation days each year, and they receive these days every
+   year on the first of January, the :guilabel:`Rate` is set to `10`, and the :guilabel:`Frequency`
+   entry is set to `Frequency (Yearly) on the (1) of (January)`. The :guilabel:`Frequency`,
+   :guilabel:`Date`, and :guilabel:`Month` fields appear.
```

Comment:
Indent example admonition to be inline with the bullet point it is an example for.
```suggestion
  .. example::
    If the employee should accrue one vacation day for every week worked, the :guilabel:`Rate` is
    set to `1`, and the :guilabel:`Frequency` entry is set to `Frequency (Weekly) on (Friday)`. Only
    the :guilabel:`Frequency` and :guilabel:`Weekday` fields appear.

    If the employee should accrue ten (10)vacation days each year, and they receive these days every
    year on the first of January, the :guilabel:`Rate` is set to `10`, and the :guilabel:`Frequency`
    entry is set to `Frequency (Yearly) on the (1) of (January)`. The :guilabel:`Frequency`,
    :guilabel:`Date`, and :guilabel:`Month` fields appear.
```